app/src/main/java/org/sil/gatherwords/EntryFragment.java

```diff
@@ -86,18 +93,14 @@ protected Word doInBackground(Long... wordIDs) {
 
         @Override
         protected void onPostExecute(Word word) {
-            View entryPage = entryPageRef.get();
+            final View entryPage = entryPageRef.get();
```

Comment:
What is the purpose for this?

Reply:
No idea, I removed it and everything still works.